src/app/backend/resource/dataselect/dataselect.go

```diff
@@ -122,7 +124,7 @@ func (self *DataSelector) Filter() *DataSelector {
 				matches = false
 				continue
 			}
-			if filterBy.Value.Compare(v) != 0 {
+			if !filterBy.Value.StartsWith(v) {
```

Comment:
Still, I disagree, but maybe it is only my impression. Check this example:

https://docs.angularjs.org/api/ng/filter/filter

As you can see they check if value contains searched string. It seems easier to use IMO. If I would like to find a pod with `kubernetes-dashboard-n32iurn23` name I can type just `n32` and it will be found. Now I need to write almost whole name.